### ISSUE TITLE
Mini HP Bars below/above players names on the side panels

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mrthuanvn/wot-xvm#280]
```
Hi,

What I was thinking is the following, the client knows how much HP each 
"visible" tank has both on your side and the enemy side.
The thing is that sometimes in the heat of battle you don't get to see and 
remember how much HP is left on each and every tank.

For example there are two last enemy tanks left in the game but they are not 
visible to you and for the life of you, you can't remember how much HP they had 
the last time you saw them.
This information is critical and it exists.

So basically you can place small HP bars next to each players names on the side 
panels and update them each time they change for visible tanks.
Once a tank has become invisible to you, the HP bar just freezes on the last 
value it was.

In this situation you can just look to the side tabs and instantly see how much 
HP is left for the final two tanks, instead of guessing.

What do you say?
```

Original issue reported on code.google.com by `DDor...@gmail.com` on 13 Feb 2013 at 2:14
